src/me/components/PinnedItems.tsx

```diff
@@ -0,0 +1,94 @@
+import React, {FC, useContext, useCallback} from 'react'
+import {capitalize} from 'lodash'
+import {
+  PinnedItemsContext,
+  PinnedItemTypes,
+  PinnedItem,
+} from 'src/shared/contexts/pinneditems'
+
+import {Context} from 'src/clockface'
+import {ComponentColor, IconFont} from '@influxdata/clockface'
+
+import './PinnedItems.scss'
+
+import {ResourceCard, ResourceList} from '@influxdata/clockface'
+
+import {useHistory} from 'react-router-dom'
+
+const PinnedItems: FC = () => {
+  const history = useHistory()
+  const followMetadataToRoute = useCallback(
+    (data: PinnedItem) => {
+      let routeToFollow = ''
+      switch (data.type) {
+        case PinnedItemTypes.Dashboard:
```

Comment:
is a PinnedItemType different than a ResourceType? Do we want them to be different?

Reply:
yes, because the concept of a pinned item transcends resources